lightkurve/lightcurve.py

```diff
@@ -953,7 +964,9 @@ def bin(self, binsize=None, bins=None, method='mean'):
         if bins is None:  # use ``binsize```
             n_bins = self.flux.size // binsize
             bin_by_array = np.arange(len(self.time))
-            bin_edges = calculate_bin_edges(bin_by_array, bins=n_bins)
+            #bin_edges = calculate_bin_edges(bin_by_array, bins=n_bins)
+            bin_edges = np.linspace(bin_by_array.min(), bin_by_array.max(),
+                        n_bins + 1, endpoint=True)
```

Comment:
@gully Did you verify that `np.linspace` and `calculate_bin_edges` do the same thing here?  It would be good to know if they don't!

Reply:
Yes, it's ultimately what the `bins=` parameter does in numpy when handed a number of bins.